presto-common/src/main/java/com/facebook/presto/common/predicate/Utils.java

```diff
@@ -40,4 +41,18 @@ public static Object blockToNativeValue(Type type, Block block)
     {
         return readNativeValue(type, block, 0);
     }
+
+    public static void checkArgument(boolean expression)
```

Comment:
You can find a lot of classes define a private `checkArgument` as a helper method. If we want to have a public method to be generally used for presto-common, this is probably not the right place. Maybe just put this into `FilterFunction` for now, or put it in a generic util class for all classes in presto-common to use.

Reply:
Thanks for the suggestion. I moved the whole Utils class from com.facebook.presto.common.predicate to com.facebook.presto.common in commit "Move FilterFunction to presto-common". Its previously existing method like `nativeValueToBlock()` was not just used in com.facebook.presto.common.predicate, but also com.facebook.presto.common.block, com.facebook.presto.spi.relation, com.facebook.presto.raptor.storage and a few other packages. So I think it's better to make it a generic util class for presto-common.

Reply:
I actually made a separate commit for moving Utils "82113c00aa Refactor Utils class in presto-common" since it involves many files. It's put as the first commit in this PR.

Reply:
Moved to FiterFunction